k8s/aat/common/ccd/data-store-api.yaml

```diff
@@ -15,7 +15,7 @@ spec:
   chart:
```

Comment:
We should also have the env variable from #4874 added to this and the prod data-store chart